### PR TITLE
JENKINS-64478 Compatibility with Jenkins >2.263

### PR DESCRIPTION
**Pull request by timja**
See https://www.jenkins.io/doc/developer/views/table-to-div-migration/

and https://www.jenkins.io/blog/2019/10/21/plugin-docs-on-github/ for docs migration

I can separate to different PRs if you wish but all commits are self contained

cc @ikedam

### CODE REVIEW

File: src/test/java/org/jenkins_ci/plugins/flexible_publish/ConfigurationTest.java [hunk 1]
```diff
@@ -430,44 +467,8 @@ public void testMatrixDisableAggregationCondition() throws Exception {
         assertEquals("anotherProject", trigger.getChildProjectsValue());
         assertEquals(Result.SUCCESS, trigger.getThreshold());
     }
-    
-    public void testNoDataBoundConstructor() throws Exception {
```
Comment (ikedam) [inline src/test/java/org/jenkins_ci/plugins/flexible_publish/ConfigurationTest.java:434]: Note for me: Create an alternate test: https://issues.jenkins.io/browse/JENKINS-64527

File: src/test/java/org/jenkins_ci/plugins/flexible_publish/ConfigurationTest.java [hunk 2]
```diff
@@ -30,6 +30,7 @@
 import java.util.Collections;
 import java.util.List;
 
+import com.gargoylesoftware.htmlunit.WebClient;
```
Comment (ikedam) [inline src/test/java/org/jenkins_ci/plugins/flexible_publish/ConfigurationTest.java:33]: I recommend `org.jvnet.hudson.test.JenkinsRule.WebClient` instead as it provides Jenkins-specific utility methods.

**Review by ikedam — CHANGES_REQUESTED** [review]
Thanks a lot for the fix and updating environments.

It would be simpler to use `<f:repeatableProperty>` as commented.
Can you see about that?
Or I can take over the work if you don't mind.

**Comment by ikedam** [comment]
Somehow, `ConfigurationTest` takes much time and some tests fails for SocketTimeout in my environment.
I'll prepare Jenkinsfile to test this on CI.

**Comment by ikedam** [comment]
Close and reopen to trigget CI.

**Comment by timja** [comment]
Please feel free to take this over, thanks!

**Comment by ikedam** [comment]
link: https://issues.jenkins.io/browse/JENKINS-64478

**Comment by ikedam** [comment]
I tried to use 2.235.1 preferring the ".1" LTS version [1], but couldn't as run-condition-1.4 with the fix for table-to-div issue [2] targets 2.235.5.

[1] https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/
[2] https://github.com/jenkinsci/run-condition-plugin/pull/22

**Review by ikedam — APPROVED** [review]

**Comment by oleg-nenashev** [comment]
Belated +1, thanks you @timja and @ikedam !

**Pull request merged**